eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -476,24 +476,24 @@ class EclairRpcClient(val instance: EclairInstance)(
     val payloadF: Future[JsValue] = responseF.flatMap(getPayload)
     payloadF.map { payload =>
       val validated: JsResult[T] = (payload \ resultKey).validate[T]
-      val parsed: T = parseResult(validated, payload)
+      val parsed: T = parseResult(validated, payload, command)
       parsed
     }
   }
 
   case class RpcError(code: Int, message: String)
   implicit val rpcErrorReads: Reads[RpcError] = Json.reads[RpcError]
 
-  private def parseResult[T](result: JsResult[T], json: JsValue): T = {
+  private def parseResult[T](result: JsResult[T], json: JsValue, commandName: String): T = {
     result match {
       case res: JsSuccess[T] =>
         res.value
       case res: JsError =>
         (json \ errorKey).validate[RpcError] match {
           case err: JsSuccess[RpcError] =>
-            logger.error(s"Error ${err.value.code}: ${err.value.message}")
-            throw new RuntimeException(
-              s"Error ${err.value.code}: ${err.value.message}")
+            val errMsg = s"Error for command=${commandName} ${instance.authCredentials.datadir.map(d => s"datadir=${d}").getOrElse("")}, ${err.value.code}=${err.value.message}"
```

Comment:
Would something like this be better?

```suggestion
 val datadirMsg = instance.authCredentials.datadir
              .map(d => s"datadir=$d")
              .getOrElse("")
            val eclairErrCodeMsg = s"code=${err.value.code}"
            val eclairErrMsg =  s"message=${err.value.message}"
            val errMsg =
              s"Error for command=$commandName $datadirMsg, $eclairErrCodeMsg, $eclairErrMsg"
```